06-read-write-plot.Rmd

```diff
@@ -125,7 +125,7 @@ Although the quality is not as spatially consistent as many official datasets, O
 Further examples of **osmdata** in action are provided in Chapters \@ref(transport) and  \@ref(location).
 <!-- Todo: Replace the above with "Chapters \@ref(location) and \@ref(gis)." when gis is online (RL) -->
 
-Finally, R packages might contain or just consist of spatial data (e.g., package **spData**).
+Finally, R packages might contain or just consist of spatial data (e.g. package **spData**).
```

Comment:
I've just spotted another issue - 'package ...' is not good English. `(for example the **spData** package)` would be better. Will change it now. Thanks for the fix!